src/Magnum/EigenIntegration/Integration.h

```diff
@@ -369,6 +370,51 @@ template<class To, std::size_t size, class T> inline To cast(const Math::Vector<
     return Math::Implementation::VectorConverter<size, T, To>::to(from);
 }
 
+
+/**
+@brief Convert Corrades StridedArrayView2D to Eigens Dynamic Matrix Type.
+*/
+template<class T> inline Eigen::Map<Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>, Eigen::Unaligned, Eigen::Stride<Eigen::Dynamic, Eigen::Dynamic>> arrayCast(const Containers::StridedArrayView2D<T>& from) {
+    using MatrixT = Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>;
```

Comment:
Could be the `RowMajor` / `ColMajor` configurable via an optional second template parameter, maybe? Not sure what should be the default tho.

Reply:
I think having it RowMajor is consistent with how it is used in Magnum (i.e.. view2D[0] is usually used as a row, on the other hand the default in Eigen is ColumnMajor)  and one can always transpose the resulting Eigen expression to get a ColumnMajor view on the data if I am not mistaken?

Reply:
I investigated and tried to support both `RowMajor` / `ColMajor` via a template parameter only to realize it doesn't really have any effect on a `Map` if you explicitly specify `Stride` as well: https://eigen.tuxfamily.org/dox/group__TutorialMapClass.html#TutorialMapTypes A `Map` can be then copied to both row- and col-major `Matrix` so unless I'm missing something it doesn't matter at all what `Matrix` type is used inside the `Map`.

So I removed the `RowMajor`, flipped order of `Stride` arguments and it resulted in various types being significantly shorter: 9379c6411a8ac95abc9bc0a7e5c2a8e95b0df4a7